parsers/prefix-by/prefix-by.parser.ts

```diff
@@ -0,0 +1,25 @@
+import { LibsType } from '../global-libs';
+import Template from '../../libs/template';
+
+export type InputDataType = Array<Record<string, any>>;
+export type OutputDataType = InputDataType;
+export type OptionsType = {
+  key?: string;
+  prefix: string; // pattern
+  types?: Array<string>;
+};
+
+export default async function (
+  tokens: InputDataType,
```

Comment:
Do you want to allow prefixes with string as input to add a comment after `to-css-custom-properties` for example?

Reply:
@ChucKN0risK  ?

Reply:
So you don't want to allow prefixing a string like the output of `to-css-custom-properties`?

Reply:
I want but I don't know how to do it.